utils/MonitoredResource.md

```diff
@@ -6,21 +6,21 @@ detect these resources.
 ## Supported Monitored Resources
 
 * GCP_GCE_INSTANCE
-  * gcp_account: The GCP account number for the instance.
+  * project_id: The identifier of the GCP project associated with this resource.
```

Comment:
We are a generic library and should not follow the names in SD monitoring. Please revert this unless good arguments.

Reply:
Done